test/addons/report-api/binding.cc

```diff
@@ -0,0 +1,26 @@
+#include <node.h>
+#include <v8.h>
+
+using v8::FunctionCallbackInfo;
+using v8::Isolate;
+using v8::Local;
+using v8::Object;
+using v8::Value;
+
+void TriggerReport(const FunctionCallbackInfo<Value>& args) {
+  Isolate* isolate = args.GetIsolate();
+
+  node::TriggerNodeReport(
```

Comment:
How do we document natives API basically? Do we just rely on header files?

Reply:
Currently, yes, with the exception of Node-API, which does have separate documentation in the Node.js API docs. I think that's okay, tbh, it's fairly common for C/C++ code to rely on header file documentation (and, in fact, I wish Node-API did that as well).

Reply:
Awesome ^^ Thanks 🙌🏼